server/server_test.go

```diff
@@ -435,7 +439,19 @@ func TestHTTPInstrumentationMetrics(t *testing.T) {
 		# TYPE tcp_connections gauge
 		tcp_connections{protocol="http"} 0
 		tcp_connections{protocol="grpc"} 0
-	`), "request_message_bytes", "response_message_bytes", "inflight_requests", "tcp_connections"))
+		
+		# HELP tcp_connections_limit The max number of TCP connections that can be accepted (0 means no limit).
```

Comment:
I'm unclear why this changed.  If it's unrelated, please put it back.

Reply:
It expanded the scope of the test, but it is indeed not related. I've put it back to previous version.